anago

```diff
@@ -554,13 +557,20 @@ announce () {
   local announcement_text=/tmp/$PROG-announce.$$
 
   ((FLAGS_nomock)) || mailto=$USER@$DOMAIN_NAME
+  if [[ -n "${FLAGS_mailto}" ]]; then
+    mailto="${FLAGS_mailto}"
+  fi
 
   if [[ -n "$PARENT_BRANCH" ]]; then
     create_branch_announcement
   else
     create_announcement
   fi > $announcement_text
 
+  ((FLAGS_yes)) \
+   || common::askyorn -e "Pausing here. Confirm announce to $mailto" \
```

Comment:
Oh I see, yes.  I'd prefer to present all info like this in the SESSION VALUES section for a complete view of the state of this session/run.  Of course that means calculating/setting $mailto globally.